cake_thief/robertwpaul/cake_thief_test.rb

```diff
@@ -0,0 +1,59 @@
+require 'minitest/autorun'
+require './cake_thief'
+
+class CakeThiefTest < Minitest::Test
+
+  def test_zero_capacity
+    cakes = [Cake.new(7, 160), Cake.new(3, 90), Cake.new(2, 15)]
+    capacity = 0
+    assert_equal 0, max_duffel_bag_value(cakes, capacity)
```

Comment:
Could the same test-case be set up using a simpler setup? For example, just one cake.

By including unnecessary test setup logic we are giving our computers a larger set of instructions for no change in the outcome.